benchmark/paddle/A2C/atari_model.py

```diff
@@ -0,0 +1,107 @@
+#   Copyright (c) 2018 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import paddle
+import parl
+import paddle.nn as nn
+import paddle.nn.functional as F
+
+
+class AtariModel(parl.Model):
+    def __init__(self, act_dim):
+        super(AtariModel, self).__init__()
+        self.conv1 = nn.Conv2D(
+            in_channels=4, out_channels=32, kernel_size=8, stride=4, padding=1)
+        self.conv2 = nn.Conv2D(
+            in_channels=32,
+            out_channels=64,
+            kernel_size=4,
+            stride=2,
+            padding=2)
+        self.conv3 = nn.Conv2D(
+            in_channels=64,
+            out_channels=64,
+            kernel_size=3,
+            stride=1,
+            padding=0)
+
+        self.flatten = nn.Flatten()
+
+        # Need to calc the size of the in_features according to the input image.
+        # The default size of the input image is 84 * 84
+        self.fc = nn.Linear(in_features=64 * 9 * 9, out_features=512)
```

Comment:
In torch-based implementation, the number of neurons for the first fc layer is  7744. https://github.com/PaddlePaddle/PARL/blob/ab1eb893a9f0ac1d5238c6a5277ea7e7c6cd1fdf/benchmark/torch/a2c/atari_model.py#L40
What causes inequality here?

Reply:
The model architecture is same to the paddle fluid version.
https://github.com/PaddlePaddle/PARL/blob/ab1eb893a9f0ac1d5238c6a5277ea7e7c6cd1fdf/examples/A2C/atari_model.py#L23-L30
And the model architecture of the torch version is:
https://github.com/PaddlePaddle/PARL/blob/ab1eb893a9f0ac1d5238c6a5277ea7e7c6cd1fdf/benchmark/torch/a2c/atari_model.py#L26-L40
The padding is different...
Maybe the model architecture of the torch-based implementation is needed to modify.

Reply:
I see. We can update the architecture of torch implementation later.